internal/auth/authn/aapgateway.go

```diff
@@ -87,10 +102,21 @@ func (a AapGatewayAuth) ValidateToken(ctx context.Context, token string) error {
 	return err
 }
 
-func (a AapGatewayAuth) GetAuthConfig() common.AuthConfig {
-	return common.AuthConfig{
-		Type: common.AuthTypeAAP,
-		Url:  a.externalGatewayUrl,
+func (a AapGatewayAuth) GetAuthConfig() *api.AuthConfig {
+	providerType := string(api.AuthProviderInfoTypeAap)
+	providerName := string(api.AuthProviderInfoTypeAap)
+	provider := api.AuthProviderInfo{
+		Name:      &providerName,
+		Type:      (*api.AuthProviderInfoType)(&providerType),
+		AuthUrl:   &a.externalGatewayUrl,
+		IsDefault: lo.ToPtr(true),
+		IsStatic:  lo.ToPtr(true),
+	}
+
+	return &api.AuthConfig{
+		DefaultProvider:      &providerType,
+		OrganizationsEnabled: lo.ToPtr(false),
```

Comment:
We did have support for organizations for AAP.
Why is this changing ?

Reply:
fixed